### ISSUE TITLE
Feature Request: rakkas should print errors while prerendering

### ISSUE THREAD

**Issue by tkamenoko** [rakkasjs/rakkasjs#44]
I got 500 errors while prerendering my blog, but error messages are not printed. `rakkas prerender` should show error details if a page returns 5xx error.

Current output looks like this:
```
rakkas: Prerendering static routes (3/3)
✓ 16 static routes prerendered.
200 dist/client/_data/80d8adc93b4/0/0/d.js
200 dist/client/_data/80d8adc93b4/1/0/d.js
200 dist/client/_data/80d8adc93b4/5/0/d.js
200 dist/client/about/index.html
500 dist/client/blog/index.html
500 dist/client/blog/page-1/index.html
500 dist/client/contact/index.html
200 dist/client/index.html
# printing more results...
```

**Comment by cyco130** [comment]
Actually Rakkas needs to print errors and warnings in a lot more situations than this. It is part of the 0.6.1 journey.

**Comment by cyco130** [comment]
Released in `0.6.1-next.0` (test with `npm i rakkasjs@next`). I'm waiting for Vite 3.1 to release `0.6.1`.

Just drop a line if it doesn't solve your problem and I'll reopen the issue.
